hack/move-upstream.sh

```diff
@@ -20,7 +25,7 @@ package="${UPSTREAM_PACKAGE:-pkg/api}"
 patch="${TMPDIR}/patch"
 rm -rf "${patch}"
 mkdir -p "${patch}"
-relativedir="../../../${repo}"
+relativedir="${UPSTREAM_DIR-../../../${repo}}"
```

Comment:
> What do you set your working dir as? Not GOPATH normative?

I keep kube and openshift in two fully discrete paths to avoid accidentally clobbering anything on godep restores.  I've also had trouble keeping them co-located when working with some sublime plugins, because they prefer the gopath in order.